components/kyma-environment-broker/internal/process/provisioning/initialisation.go

```diff
@@ -243,3 +260,35 @@ func (s *InitialisationStep) launchPostActions(operation internal.ProvisioningOp
 
 	return s.operationManager.OperationSucceeded(operation, msg)
 }
+
+func (s *InitialisationStep) createTagsForRuntime(operation internal.ProvisioningOperation) ([]*avs.Tag, internal.ProvisioningOperation, time.Duration, error) {
+	instance, err := s.instanceStorage.GetByID(operation.InstanceID)
+	if err != nil {
+		return []*avs.Tag{}, operation, 10 * time.Second, err
+	}
+
+	status, err := s.provisionerClient.RuntimeStatus(instance.GlobalAccountID, operation.RuntimeID)
+	if err != nil {
+		return []*avs.Tag{}, operation, 1 * time.Minute, err
+	}
+
+	result := []*avs.Tag{
+		&avs.Tag{
+			Content:      ptr.ToString(status.RuntimeConfiguration.ClusterConfig.Name),
```

Comment:
"Name, Seed, Region" is the same data that we sent to the provisioner during the create runtime operation?

Reply:
Name - Gardener shoot name, it is currently generated in Provisioner
Seed - this field is optional, we do not send it to provisioner (and provisioner also does not set it, it's done by Gardener right now - bug reported https://github.com/kyma-project/control-plane/issues/363)
Region - that's right, it's the same data we sent to provisioner, unless provisioner does some additional mapping

Reply:
The name will be generated from KEB side after merge [PR](https://github.com/kyma-project/control-plane/pull/370), so there is only Seed generated by the gardener, it's a pity we could have avoided one request to an external service.

Reply:
f2f decision: we'll leave it like this at the moment, after #363 is somehow resolved by provisioner we can change the logic to use KEB internal data